scripts/provisioning/configcmd.py

```diff
@@ -159,7 +159,7 @@ def configure_openldap_replication(self):
         handler = SimpleProcess(cmd)
         stdout, stderr, retcode = handler.run()
         self.logger.info(f'output of setupReplicationScript.sh: {stdout}')
-        os.remove("hosts_list_file.txt")
+        #os.remove("hosts_list_file.txt")
```

Comment:
1)Specific file location for 'hosts_list_file.txt' instead of current directory
2)Rename it to ldap_hosts_list_file.txt
3)Adding validator for 'data>private_fqdn' paramete